src/tests/components/CollectionIcons.test.ts

```
@@ -0,0 +1,43 @@
+import { render, fireEvent } from "@testing-library/svelte";
+import { describe, it, expect, vi } from "vitest";
+import CollectionIcons from "../../components/collections/CollectionIcons.svelte";
+
+describe("CollectionIcons", () => {
+    it("calls onRemoveIcon when the remove button is clicked", async () => {
+        const mockOnRemoveIcon = vi.fn();
+        const mockCollection = {
+            id: "collection-id",
+            icons: [{ id: "icon1", label: "Icon 1", base64Thumbnail: "data:image/png;base64,..." }]
+        } as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)